src/common/config.ts

```diff
@@ -10,4 +10,15 @@ config.INTEGRATION_ID = config.INTEGRATION_ID.trim();
 config.CLUSTER_NAME = config.CLUSTER_NAME || 'Default cluster';
 config.IMAGE_STORAGE_ROOT = '/var/tmp';
 
+/**
+ * Important: we delete the following env vars because we don't want to proxy requests to the Kubernetes API server.
+ * The Kubernetes client library would honor the NO/HTTP/HTTPS_PROXY env vars.
+ */
+config.HTTPS_PROXY = process.env['HTTPS_PROXY'];
```

Comment:
I wonder if we can do something a bit smarter - so customers insert values like `HTTPS_PROXY` but we pass them to the monitor as an env var called `SNYK_HTTPS_PROXY` (only in helm charts). so we will get it only as `config.HTTPS_PROXY` but it won't be part of the env vars, so no need to delete them

Reply:
Well that was my initial proposal 😛

Reply:
I mean that the customers will still have the exact same interface of inserting `HTTPS_PROXY`, but we will map it to a `SNYK_HTTPS_PROXY` env var in the container

Reply:
Just to make sure I understand, we will use `http_proxy`, `https_proxy`, etc. in the Helm chart, just in the `deployment.yaml` we will use `SNYK_HTTP_PROXY` for the name of the env var?

Reply:
that's only my suggestion. what we actually want is customers to set `http_proxy` etc and for that to be "injected" to our config, right? 
not saying that's the best way to go, just a suggestion